libs/core-ui/src/lib/components/InteractiveLegend.tsx

```diff
@@ -83,4 +88,27 @@ export class InteractiveLegend extends React.PureComponent<IInteractiveLegendPro
       </div>
     );
   }
+
+  private getColorBoxCss = (
```

Comment:
Could you add a unit test for this function?

Reply:
Moved to util file and added unit test thanks! 
![image](https://user-images.githubusercontent.com/76190371/186545871-7c49d1c2-e85b-4a0c-b014-3a93996bcd56.png)